contracts/bridge/BridgeTestAnotherVersion.sol

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity 0.5.6;
```

Comment:
now, can we use later version of solidity?

Reply:
oh, it might require more changes to other contracts. Let's do this later.

Reply:
We are planning to update the solidity version in the build base docker image that is used by Circl ci. 
Can you update this version to 0.8.x  or make json file with compiled data?
You better check https://github.com/klaytn/klaytn/pull/1189/files first to understand my request.

Reply:
I see. I was working on implementing the bridge interface checker and KIP contracts for value transfer on SC. Let me go this up first. If you have another consideration for the contract, please let me know early.

Reply:
Once the reconstruction of the official contract library for Klaytn is done, I'll import them.